.Rbuildignore

```diff
@@ -16,3 +16,4 @@ cran-comments.md
 vignettes/figure
 ^CONDUCT\.md$
 ^cran-comments\.md$
+^codemeta\.json$
```

Comment:
Looks like @sckott may have since updated `.Rbuildignore`; somehow your PR has a merge conflict on this file.  You should just be able to `git pull https://github.com/ropensci/rbison.git` to pull in his changes and then resolve the conflict manually and push back up the merge, though might be easier for @sckott to resolve on GitHub directly.

Reply:
yeah, was this one https://github.com/ropensci/rbison/commit/26809194eda436cfe305e4225edf0231bcca228d